xarray/core/dataarray.py

```diff
@@ -1474,6 +1474,41 @@ def thin(
         """Return a new DataArray whose data is given by each `n` value
         along the specified dimension(s).
 
+        Examples
```

Comment:
Is the one line description above clear enough?

Reply:
Yes!